cmake/setup_cached_variables.cmake

```diff
@@ -331,7 +331,7 @@ unset(ENV{NVCCFLAGS})
 ########################################################################
 
 option(DEAL_II_WITH_64BIT_INDICES
-  "If set to ON, then use 64-bit data types to represent global degree of freedom indices. The default is to OFF. You only want to set this to ON if you will solve problems with more than 2^31 (approximately 2 billion) unknowns. If set to ON, you also need to ensure that both Trilinos and/or PETSc support 64-bit indices."
+  "If set to ON, then use 64-bit data types to represent global degree of freedom indices. The default is to OFF. You only want to set this to ON if you will solve problems with more than 2^31 (approximately 2 billion) unknowns. If this is set to ON then Trilinos must also be configured to support 64-bit indices."
```

Comment:
I'm not a fan of bare "this" as in "If this is set to ON" because it never seems to be quite clear what "this" is actually referring to. Either say "this parameter" or "this flag", or some something like this.

Reply:
Let us simply remove the comment for Trilinos. There is in fact only one official way to configure Trilinos and this is with 64bit signed ordinals.